packages/js/src/Modules/Verto/webrtc/helpers.ts

```diff
@@ -373,6 +373,32 @@ const sdpBitrateHack = (
   return lines.join(endOfLine);
 };
 
+const sdpBitrateASHack = (
+  sdp: string,
+  bandwidth_kbps: number
+) => {
+  let modifier = 'AS';
+  let bandwidth = bandwidth_kbps;
+
+  if (
+    navigator.userAgent.match(/firefox/gim) &&
+    !navigator.userAgent.match(/OPR\/[0-9]{2}/gi) &&
+    !navigator.userAgent.match(/edg/gim)
+  ) {
+    modifier = 'TIAS';
+    bandwidth = (bandwidth_kbps >>> 0) * 1000;
```

Comment:
Can you add a constant for these magic numbers to explains what they do, please?

For example:

```
(bandwidth_kbps >>> 0) * 1000;

(bandwidth_kbps >>> 0) * ONE_SECOND;

```

It is only an example I don't know what is this value for.